tower-backend/src/main/groovy/io/seqera/tower/service/UserServiceImpl.groovy

```diff
@@ -143,6 +143,23 @@ class UserServiceImpl implements UserService {
         UserRole userRole = new UserRole(user: user, role: role)
         userRole.save()
 
+        // Try to get a gravatar avatar URL
+        try {
+            String emailHash = email.trim().toLowerCase().md5()
+            String gravatarURL = "https://www.gravatar.com/avatar/${emailHash}"
+            def gravatarConnection = new URL("${gravatarURL}?d=404").openConnection().with {
+                requestMethod = 'HEAD'
```

Comment:
What is use for `d` parameter? image size?

Reply:
No - by default, gravatar returns a generic image if the email isn't found:

![default](https://www.gravatar.com/avatar/00000000000000000000000000000000)

Adding the `?d=404` tells it to give a 404 file not found error instead, if the email isn't found.

https://en.gravatar.com/site/implement/images/

Reply:
..you can specify image size with `s=[width-in-px]` though.